api/swagger-spec/storage.k8s.io_v1beta1.json

```diff
@@ -675,7 +675,7 @@
      },
      "metadata": {
       "$ref": "v1.ObjectMeta",
-      "description": "Standard object's metadata. More info: http://releases.k8s.io/HEAD/docs/devel/api-conventions.md#metadata"
```

Comment:
I think the script I mentioned was in gh-pages branch (which we have now deleted).

Yes this PR is fine. But we need a way to replace `https://github.com/kubernetes/community/blob/master/contributors/devel/api-conventions.md#metadata` by `https://github.com/kubernetes/community/blob/release-1.6/contributors/devel/api-conventions.md` while pushing these docs to k8s.io, right?
Can do that in update-imported-docs.

Feel free to file a separate issue if you agree.